frontend/discourse/app/models/bookmark.js

```diff
@@ -99,6 +99,10 @@ export default class Bookmark extends RestModel {
 
   @discourseComputed("highest_post_number", "url")
   lastPostUrl(highestPostNumber) {
```

Comment:
Maybe `lastPostUrl` doesn't quite make sense anymore for a bookmark and we should refactor this to instead use a more "generic" term that works for "all types" of bookmarks?